pandas/tests/reshape/test_pivot_multilevel.py

```diff
@@ -190,3 +190,35 @@ def test_pivot_list_like_columns(
         expected_values, columns=expected_columns, index=expected_index
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_pivot_multiindexed_rows_and_cols():
+    # GH 36360
```

Comment:
do we have appropraite tests if index OR columns are a MI and the other is not?

Reply:
We do (in this file and in `test_pivot.py`)

What existing tests don't cover is the case when `pivot_table` produces a result with multiple MutliIndexed columns - added here